resalloc/client.py

```diff
@@ -38,12 +38,16 @@ class Connection(object):
     def __init__(self, conn):
         self.connection = C_XMLRPC(conn)
 
-    def newTicket(self, tags=None):
+    def newTicket(self, tags=None, sandbox=None):
         if tags == None:
             raise ResallocClientException("no tags specified")
 
         t = Ticket(connection=self.connection)
-        t.id = self.connection.takeTicket(tags)
+        args = [tags]
+        if sandbox:
+            args.append(sandbox)
+
+        t.id = self.connection.takeTicket(*args)
```

Comment:
```suggestion
        t.id = self.connection.takeTicket(tags=tags, sandbox=sandbox)
```

Condition is not needed. takeTicket by default gets None.

Reply:
This goes through xmlrpc, and `sandbox=` named argument doesn't work there.  If you have an idea how to make this work, let me know.

Reply:
I am not able to find out `takeTicket` function in xmlrpc.client.ServerProxy https://github.com/python/cpython/blob/3.8/Lib/xmlrpc/client.py#L1389

Reply:
Let's leave it as you propose.

Reply:
It is xmlrpc call to ServerAPI.